prac_07/myguitars.py

```diff
@@ -15,4 +15,5 @@ def main():
         print(guitar)
 
```

Comment:
Your file lost quite a lot of functions, like asking the user to input their guitar information, your display function with a lack of vintage identification, and also writing back to the CSV function.